aireview/justfile

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env just --justfile
 
```

Comment:
**documentation**: Removal of dotenv-load may cause issues if environment variables are needed. Consider documenting the change or ensuring variables are provided through another method.